src/auto/event.rs

```diff
@@ -1,162 +0,0 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir)
```

Comment:
Not sure why `event.rs` gets deleted, but it doesn't look right.

Any ideas? cc @alatiera @GuillaumeGomez

Reply:
There'es api missing from auto/surface and auto/seat as well, need to look into